### ISSUE TITLE
please push newest docker images

### ISSUE THREAD

**Issue by 0seastar0** [human-pangenomics/hpp_production_workflows#6]
Hi!

I ran the standard_haploid_qc workflow and saw that it was using a really old version of minimap2 in the asmgene call. 

The asmgene task (here: https://github.com/human-pangenomics/hpp_production_workflows/blob/master/QC/wdl/tasks/asmgene.wdl), uses the docker image "tpesout/hpp_minimap2:latest"
If Iyou pull and start that docker image, and run minimap2 --version:
$ docker run -it --rm tpesout/hpp_minimap2:latest /bin/bash
root@9826e39b4f73:/data# minimap2 --version
2.17-r941

However, the docker build file (here: https://github.com/human-pangenomics/hpp_production_workflows/blob/master/QC/docker/minimap2/Dockerfile) points at a much newer (1 release back) version.

We think @tpesout Trevor forgot to update the docker images with the latest build,  maybe did it on a local machine but didn't publish the image.

Thanks,
Sara and Bruce @bkmartinjr

**Comment by 0seastar0** [comment]
Hey @juklucas @tpesout 

Any updates on this issue? We are blocked right now. Thank you!

-Sara

**Comment by tpesout** [comment]
Hi @0seastar0 I just pushed the latest on master to `tpesout/hpp_minimap2:latest`.  

I see that it's not documented, but it's possible for anyone to build the image locally and push it to your personal repository, then specify that docker image in the workflow.  I believe that the command is something like

```
make repository=<your_repo>
make push repository=<your_repo>
```

**Comment by bkmartinjr** [comment]
@tpesout - thank you!  As a point of future clarification, will dockerhub normally be up to date with the workflows, or we will need to plan to always build images locally?  (IMHO, it is much better for downstream users if they can just assume the github workflows and dockerhub images are in sync)

**Comment by tpesout** [comment]
Yeah, it should normally be up to date, and I suspect that it may have been up to date on @juklucas's repo.

We're planning on transitioning the default repository to an official hpp/hprc repo and also including specific version references in the WDL defaults, but just have had other more pressing work to do.

**Comment by bkmartinjr** [comment]
Fantastic - this sounds like an excellent target.  Appreciate your work in this!
